.github/workflows/dependabot-batch-updater.yml

```
@@ -0,0 +1,18 @@
+name: Check if dependencies can be updated and create a pull request with updated ones.
+on:
+  schedule:
+    # Run once a week at 5:00
+    - cron:  '0 5 * * */7'
```

should we make it on a fixed day (e.g. every Wednesday):
```suggestion
    - cron:  '0 5 * * 3'
```

(I always rely on [crontab.guru](https://crontab.guru/#0_5_*_*_3) for that kind of stuff...)

---

oh wow, that's indeed a cool website